index.html

```diff
@@ -9222,6 +9225,256 @@ <h3 id="example-2">
+            <dfn id="#envelope-rate">Computing the envelope rate</dfn> is done
+            by applying a function to the ratio of the <var>compressor
+            gain</var> and the <var>detector average</var>. User-agents are
+            allowed to choose the shape of the attack and release curve.
+            However, they MUST respect the following constraints:
+          </p>
+          <ul>
+            <li>The input to the curve MUST be the ratio of the <var>compressor
+            gain</var> and the <var>detector average</var>.
+              <div class="note">
+                When attacking, this number less than or equal to 1, when
+                releasing, this number is strictly greater to 1.
+              </div>
+            </li>
+            <li>The attack curve MUST be a continuous, monotonically increasing
+            function in the range \([0, 1]\).
+            </li>
+            <li>The release curve MUST be a continuous, monotonically
+            decreasing function that is always greater to 1.
+            </li>
+          </ul>
+          <div class="note">
+            It is allowed, for example, to have a compressor that performs an
+            <em>adaptative release</em>, that is, releasing faster the harder
+            the compression, or to have curves for attack and release that are
+            not of the same shape.
+          </div>
+          <p>
+            Applying a <dfn>compression curve</dfn> to a value means computing
+            the value of this sample when passed to a function, and returning
+            the computed value. This function MUST respects the following
+            characteristics:
+          </p>
+          <ol>
+            <li>This function is the identity up to the value of the linear
+            <code>threshold</code> (i.e., \(f(x) = x\)).
+            </li>
+            <li>From the <code>threshold</code> up to the <code>threshold +
+            knee</code>, User-Agents can choose the curve shape. The whole
+            function MUST be monotonically increasing and continuous except
+            when the <code>knee</code> parameter is 0, in which case a
+            discontinuity happens at the treshold.
+              <div class="note">
+                If the <code>knee</code> is 0, the
+                <a>DynamicsCompressorNode</a> is called a hard-knee compressor.
+              </div>
+            </li>
+            <li>This function returned is linear, based on the ratio, after the
+            <code>threshold</code> and the soft knee (i.e., \(f(x) =
+            \frac{1}{ratio} \cdot x \)).
+            </li>
+          </ol>
+          <p>
+            Converting a value \(v\) in <dfn id="linear-to-decibel">linear gain
+            unit to decibel means</dfn> executing the following steps:
+          </p>
+          <ol>
+            <li>If <var>v</var> is equal to zero, return -1000.
+            </li>
+            <li>Else, return \( 20 \times \log_{10}{v} \)
```

Comment:
`\times` not really needed, but ok.  (Might need `\,` or something if you do remove the times, so that there's a tiny space between 20 and log.)